spec/system/visit_change_password_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'ChangePassword', type: :system do
+  before(:each) do
+    login_gds_user
+  end
+
+  it 'shows form' do
+    visit profile_change_password_path()
+    expect(page).to have_content 'Old password'
```

Comment:
the text shouldn't be hardcoded but use the locale instead, e.g. `t('password.change_password_btn')`